src/actions/profilesActions.ts

```diff
@@ -162,7 +163,7 @@ export function updateProfile(accountAddress: string, name: string, description:
 export type FollowItemAction = IAsyncAction<"FOLLOW_ITEM", { accountAddress: string }, { type: FollowType; id: string; isFollowing: boolean}>;
 
 export function toggleFollow(accountAddress: string, type: FollowType, id: string) {
-  return async (dispatch: any, _getState: any) => {
+  return async (dispatch: any, _getState: any): Promise<boolean> => {
```

Comment:
All of these function params should be `(dispatch: Redux.Dispatch<any, any>, _getState: () => IRootState)`

Reply:
Thanks!